src/ItemProgress.h

```diff
@@ -50,37 +51,43 @@ class ItemProgress : public MenuItem {
     /**
      * @brief Increments the progress of the list.
      */
-    void increment() override {
-        if (progress >= MAX_PROGRESS) return;
+    void increment() {
+        if (progress >= MAX_PROGRESS) {
+            return;
+        }
         progress += stepLength;
     }
```

Comment:
[Optional]
Now that you've made this easier to test (thanks 😊), consider adding a few unit tests for this action.
Same for:
- [ ] decrement
- [ ] setProgress

Reply:
Agree, but let me do it on next pull request.
We still on separated branch, so we can split to different changes.